src/AaveCurator/AaveCurator.sol

```diff
@@ -0,0 +1,301 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import {console2} from 'forge-std/Test.sol';
+
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {Ownable} from 'solidity-utils/contracts/oz-common/Ownable.sol';
+import {Initializable} from 'solidity-utils/contracts/transparent-proxy/Initializable.sol';
+import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+
+import {IWeth} from './interfaces/IWeth.sol';
+import {IWstEth} from './interfaces/IWstEth.sol';
+import {ILido} from './interfaces/ILido.sol';
+import {IMilkman} from './interfaces/IMilkman.sol';
+import {IRocketPoolDeposit} from './interfaces/IRocketPoolDeposit.sol';
+
+contract AaveCurator is Initializable, Ownable {
```

Comment:
in essence what you want to archive (i think) is https://github.com/bgd-labs/solidity-utils/blob/main/src/contracts/access-control/OwnableWithGuardian.sol#L7

where:
- owner is the short exec
- guardian is what you call the manager